packages/wallet/ui/src/components/Proposal.jsx

```diff
@@ -15,6 +15,9 @@ const OfferEntryFromTemplate = (
 ) => {
   const value = BigInt(stringifiedValue);
   const purse = purses.find(p => p.pursePetname === pursePetname);
+  if (!purse) {
```

Comment:
We might want to just throw here. I think there's an error boundary around it

Reply:
Then we'd get a flash of an error until the purse resolved, which looked bad in my testing.

Reply:
The capitalization makes `OfferEntryFromTemplate` look like a component, and then it's surprising for a component to decide not to render itself.

consider naming this `makeOfferEntryForTemplate`. The `make` to show that it's not a component itself and `For` to acknowledge that there may not be one for the template.